### PR TITLE
Example wiping domain updates

### PR DESCRIPTION
**Pull request by m-j-mcdonald**
This is an example of how I would clean up the current wiping domain code. I have not run this code due to issues in getting super cloud configured, but these changes are at least a good first step to getting things running.

A few points to note:
1. I would never enforce RobotAt in the optimization unless absolutely necessary (e.g., for a simple MoveTo action that can't infer positions from relevant objects). They can be useful for the task planner sometimes, so leave them with start time > end time in the domain spec. Your current error I think is coming from enforcing this constraint at the end timestep (and ending up in conflict with what the pose suggester is trying to do).
2. Pay close attention to how you set up the pose suggester. The end pose symbol is something you explicitly provide the values for; for this action you cannot set the pose using targ.right_ee_pos because targ.right_ee_pos is something that the pose suggester would have to determine (unless you hard-code the end pose to be in contact with the table, but this is bad design). Instead you should compute a point on the table and then pass to the ik solver.

Basically, I think something that will help is to think of RobotPose symbols as being there mostly for legacy reasons; there are cases where you need to use them but not for this domain. Instead you should be inferring poses directly from the environment configuration.

### CODE REVIEW

**Comment by m-j-mcdonald** [comment]
I'd also like to note that the current definition of InContactRobotTable is close to how you want it but not quite. The grasp_pt for a Box geometry is 0.03 below the top of the Box (see items.py for this); this is because to grasp something you need to have it inside the gripper. For the table, you want to be in contact with the surface, hence you need to compute a different value for rel_pt. I would go with `self.rel_pt = np.array([0, 0, params[1].geom.height])`
